lib/improver/wind_downscaling.py

```diff
@@ -89,10 +89,10 @@ def __init__(self, u_href, h_ref, z_0, mask):
         self.z_0 = z_0
         self.mask = mask
 
-    def calc_ustar(self):
+    def process(self):
```

Comment:
In the docstring of the `__init__`, I think that the type of the variables should be updated to np.ndarray. The description then can be updated to say "A 2D array of floats for the wind speed at h_ref". For example:
```python
Args:
    u_href (np.ndarray): 
        A 2D array of floats for the wind speed at h_ref.
```
The `comments` section could be updated to be a `Notes` section, as this is supported by [Sphinx](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html#docstring-sections) as shown in [readthedocs](http://improver.readthedocs.io/en/latest/improver.wind_downscaling.html#module-improver.wind_downscaling).